integration/pom.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>beadledom-integration</artifactId>
+  <packaging>pom</packaging>
+  <name>Beadledom Integration</name>
+
+  <parent>
+    <artifactId>beadledom-parent</artifactId>
+    <groupId>com.cerner.beadledom</groupId>
+    <version>3.3-SNAPSHOT</version>
+  </parent>
+
+  <modules>
+    <module>api</module>
+    <module>service</module>
+    <module>client</module>
+    <module>test</module>
+  </modules>
+
+  <properties>
+    <jetty.version>9.4.18.v20190429</jetty.version>
```

Comment:
Looks like there is a newer version: 9.4.19.v20190610

Reply:
There is: https://github.com/cerner/beadledom/pull/149/commits/17f663e0b1f0011fdba37d921c18f6e2e30788bc